src/Symfony/Component/Security/Core/Authorization/Voter/Vote.php

```diff
@@ -11,13 +11,22 @@
 
 namespace Symfony\Component\Security\Core\Authorization\Voter;
 
+/**
+ * A Vote is returned by a Voter and contains the access (granted, abstain or denied). It can also contains a reason
```

Comment:
```suggestion
 * A Vote is returned by a Voter and contains the access (granted, abstain or denied).
 * It can also contains a reason explaining the vote decision.
```